tools/online_editor/src/index.ts

```diff
@@ -351,8 +345,12 @@ import slint_init, * as slint from "@preview/slint_wasm_interpreter.js";
     }
 
     if (component !== undefined) {
-      const instance = component.create(canvas_id);
-      instance.show();
+      if (current_preview_instance === null) {
```

Comment:
Please consider to do `current_preview_instance == null`. That catches both `undefined` and `null`.

Reply:
But `current_preview_instance` cannot be undefined, hence the check for strict null.

Reply:
It's not important (none of my comments were, otherwise I would not have approved the PR).

It's just one character less to type and catches all cases that can theoretically (it can not in your current code!) go wrong for no loss.